api/lib/reducers/emoneyV0-reducers.js

```diff
@@ -13,9 +13,11 @@ async function totalBackedValueReducer(
   const lunieCoin = reducers.coinReducer(totalBackedValue, coinLookup)
 
   // The total net EUR value of the token's total supply
-  const fiatValue = BigNumber(lunieCoin.amount)
-    .times(exchangeRates[lunieCoin.denom][aggregatingCurrency])
-    .toNumber()
+  const fiatValue = exchangeRates
```

Comment:
this is useful if we are developing and don't have the emoney env variables in our `.env`